ignite/cmd/network_chain_join.go

```diff
@@ -33,8 +33,32 @@ func NewNetworkChainJoin() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "join [launch-id]",
 		Short: "Request to join a network as a validator",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainJoinHandler,
+		Long: `The "join" command is used by validators to send a request to join a blockchain.
+The required argument is a launch ID of a blockchain. The "join" command expects
+that the validator has already setup a home directory for the blockchain and has
+a gentx either by running "ignite network chain init" (recommended) or
```

Comment:
I would not say it's recommended to run `init`. It depends on the validator needs